src/components/expanded-state/custom-gas/GweiInputPill.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { useCallback } from 'react';
-// @ts-expect-error
```

Comment:
Same as above, this won't be a diff once https://github.com/rainbow-me/rainbow/pull/4157 is merged.